MainWindow.h

```diff
@@ -108,11 +114,15 @@ class MainWindow : public QMainWindow {
 
   std::unique_ptr<buffers::Project> _project;
   QPointer<RecentFiles> _recentFiles;
+  
+  std::unique_ptr<EventData> _event_data;
+  egm::EGM egm;
 
   void openSubWindow(buffers::TreeNode *item);
   void readSettings();
   void writeSettings();
   void setTabbedMode(bool enabled);
+  static QFileInfo getEnigmaRoot();

```

Comment:
This is a horrible function name, also add comments to clarify what it does.

Reply:
It gets the root of enigma :/ I dunno what else you'd name it.

Reply:
ENIGMA root could be anything, is my point, it could be interpreted as meaning the protobuf/tree root or etc. Probably `GetEnigmaFileSystemRoot` would be miles better.

Reply:
none of the things you listed are enigma